### DISCUSSION TITLE
Detalhes de configuração

### DISCUSSION THREAD

**Discussion by Ramalfi** [sircharlo/meeting-media-manager#86] [Q&A] [answered]
Primeiramente muito obrigado pela excelente ferramenta.
Gostaria de uma ajuda para explicar melhor o que cada configuração faz por exemplo, o que é a configuração de  "habilitar conversão de midia externa"?

**Comment by sircharlo** [comment, marked as answer]
Hi! I don't speak Portuguese, so forgive my machine-translated answer. 
 ___
Allow me to answer with the new upcoming update in mind, as a few features have changed somewhat.

**Enable button to play Kingdom songs before and after meetings**: Shows a button which will shuffle Kingdom songs before and after meetings. This could be useful for when we return to Kingdom Halls.

**Convert downloaded media files to MP4**: This will automatically convert picture and audio files to MP4 format, in order to be able to use the feature in Zoom to share MP4 files directly, without sharing a part of your screen. Sharing an MP4 file directly in Zoom results in better sound and audio quality for those in attendance at the meeting.

**Offer to import and convert additional media files**: This feature will allow you to add additional media files. Files added at this step are also converted into MP4, if the previous setting is enabled. You can choose to add:
- a song, for example for a public speech (choose the song number and it will be downloaded for you)
- the media files from a JWPUB file, such as the S-34
- any other media file from your computer
___

Permitam-me que responda com a nova actualização em mente, uma vez que algumas características mudaram um pouco.

**Habilitar botão para tocar canções do Reino antes e depois das reuniões**: Mostra um botão que baralhará as canções do Reino antes e depois das reuniões. Isto pode ser útil para quando regressarmos aos Salões do Reino.

**Converter ficheiros multimédia descarregados para o formato MP4**: isto converterá automaticamente ficheiros de imagem e áudio para o formato MP4, de modo a poder utilizar a funcionalidade no Zoom para partilhar ficheiros MP4 directamente, sem partilhar uma parte do seu ecrã. Partilhar um ficheiro MP4 directamente no Zoom resulta numa melhor qualidade de som e áudio para os participantes na reunião.

**Oferta de importação e conversão de ficheiros multimédia adicionais**: Esta funcionalidade permitir-lhe-á adicionar ficheiros de media adicionais. Os ficheiros adicionados nesta etapa são também convertidos em MP4, se a definição anterior estiver activada. Pode escolher adicionar
- uma canção, por exemplo para um discurso público (escolha o número da canção e esta será descarregada para si)
- ficheiros multimédia de um ficheiro JWPUB, tal como o S-34
- qualquer outro ficheiro multimédia do seu computador

**Reply by sircharlo** [reply]
The new update is live, with updated documentation as well. I suggest you check it out!
___
A nova actualização está agora online, juntamente com documentação actualizada. Sugiro que a consulte!

**Reply by sircharlo** [reply]
"Fixed" since https://github.com/sircharlo/jw-meeting-media-fetcher/commit/9b236a82806a5a5f148b6808e5c3f86451759c90
